tests/collections/asr/numba/rnnt_loss/test_rnnt_pytorch.py

```diff
@@ -494,5 +498,31 @@ def test_case_randomized_act_label(self, device):
             assert np.allclose(pt_grads, ag_grads, rtol=1e-2), "multi-blank gradient mismatch."
 
 
+class TestTDTLoss:
+    @pytest.mark.unit
```

Comment:
I wonder if we can use pytorch loss version to construct a hard coded fixed input/output tensor and compute loss so that we can do a minimal test with fixed input output.

Reply:
Done.

Reply:
Thanks !